internal/messenger/email/email.go

```diff
@@ -88,18 +98,49 @@ func (e *Emailer) Name() string {
 	return emName
 }
 
+func emailAddressMatchesAllowlist(emailAddress string, allowedFromAddresses []string) bool {
```

Comment:
I would have expected this to be a method of the Server.
Would make the call shorter as you save one very long parameter name.

Reply:
that's a very good idea :)